server/controllers/finance/reports/reportAccounts/index.js

```diff
@@ -67,54 +62,49 @@ function document(req, res, next) {
 /**
  * @function queryReportAccount
  * This feature select all transactions for a specific account
-*/ 
-function queryReportAccount(accountId, source){
+*/
+function queryReportAccount(accountId, source) {
   source = parseInt(source);
+  let sql;
 
-  if(source === 1){
-
-    let sql = `
+  if (source === 1) {
+    sql = `
       SELECT general_ledger.trans_id, BUID(general_ledger.entity_uuid) AS entity_uuid, general_ledger.description,
       general_ledger.trans_date, general_ledger.debit_equiv as debit, general_ledger.credit_equiv as credit
       FROM general_ledger
       WHERE general_ledger.account_id = ?
       GROUP BY general_ledger.trans_id
-      ORDER BY general_ledger.trans_date ASC;`;
-
-    return db.exec(sql, [accountId]);
-      
-  } else if(source === 2) {
-
-    let sql = `
+      ORDER BY general_ledger.trans_date ASC;
+    `;
+  } else if (source === 2) {
+    sql = `
       SELECT posting_journal.trans_id, BUID(posting_journal.entity_uuid) AS entity_uuid, posting_journal.description,
       posting_journal.trans_date, posting_journal.debit_equiv as debit, posting_journal.credit_equiv as credit
       FROM posting_journal
       WHERE posting_journal.account_id = ?
       GROUP BY posting_journal.trans_id
-      ORDER BY posting_journal.trans_date ASC;`;
-
-    return db.exec(sql, [accountId]);
-
-  } else if(source === 3){
-    let sql = `
+      ORDER BY posting_journal.trans_date ASC;
+    `;
+  } else if (source === 3) {
```

Comment:
Not in this pull request - as you mentioned this report could use a refactor/ audit. Could you make an issue to track this?